src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -4127,6 +4151,7 @@ namespace ReportSizingManager {
             DesExitTemp =
                 max(FinalSysSizing(SysNum).CoolSupTemp, AvgZoneTemp - ZoneCoolLoadSum / (StdRhoAir * CpAir * FinalSysSizing(SysNum).DesCoolVolFlow));
             DesFlow = FinalSysSizing(SysNum).DesCoolVolFlow;
+            DesExitHumRat = Psychrometrics::PsyWFnTdbRhPb(DesExitTemp, 0.9, DataEnvironment::StdBaroPress, "GetCoilDesFlowT");
```

Comment:
Why 90% RH here?

Reply:
We don't know how close the supply air condition is to saturation at this point. Could choose 95% maybe but probably doesn't make much of a difference. @EnergyArchmage, any input to the reasoning of this choice?

Reply:
@rraustad @EnergyArchmage Not clear in the discussions above if the 5ZoneAirCooled numbers are with Chicago weather (as it should be) or with Golden weather (like CI runs for whatever reason, won't change that until after release).  With Golden weather, the new warning appears and design conditions change, as well as the size, even though it's Chicago design days but with different altitude - that's all? 

Bigger question is: With Chicago weather, there is no coil sizing warning (develop or this branch) yet the sizing results for the Main Cooling Coil have changed. Wasn't expecting that.

Reply:
When the water coil sizes capacity it includes fan heat at the inlet/outlet for draw/blow through configurations. This isn't getting propagated to the DataDesOutlet/InletAirTemp variables.